osig/settings.py

```
@@ -235,15 +254,33 @@
     "workers": 4,
     "max_attempts": 2,
     "redis": env("REDIS_URL"),
+    "error_reporter": {},
 }
 
+
+def extract_from_record(logger, name, event_dict):
```

Ruff (0.14.0)

Unused function argument: `logger`

(ARG001)